src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationOptions.cs

```diff
@@ -22,7 +22,7 @@ public CookieAuthenticationOptions()
             ReturnUrlParameter = CookieAuthenticationDefaults.ReturnUrlParameter;
             ExpireTimeSpan = TimeSpan.FromDays(14);
             SlidingExpiration = true;
-            CookieSameSite = SameSiteMode.Strict;
+            CookieSameSite = SameSiteMode.Lax;
```

Comment:
Can you put a comment in indicating why we're using `lax`? Even just a link to the issue would be sufficient. Just something so that somebody doesn't come along 6 months later and decide to change it :)